hip-service/wwwroot/swagger.yaml

```diff
@@ -171,3 +329,61 @@ components:
           type: string
       xml:
         name: Error
+
+    LinkReference:
+      type: object
+      required:
+      - referenceNumber
+      - authenticationType
+      properties:
+        referenceNumber:
+          type: string
+        authenticationType:
+          type: string
+          enum: ['DIRECT', 'MEDIATED']
+        meta:
+          type: object
+          $ref: '#/components/schemas/meta'
+      xml:
+        name: LinkReference
+
+
+    meta:
```

Comment:
do we need another attr, explaining how long the OTP is valid for mediated transfer? like "validity" (in secs)